docs/topics/clusterdefinitions.md

```diff
@@ -73,6 +73,18 @@ $ aks-engine get-versions
 | maximumLoadBalancerRuleCount     | no                        | Maximum allowed LoadBalancer Rule Count is the limit enforced by Azure Load balancer. Default is 250                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | kubeProxyMode                    | no                        | kube-proxy --proxy-mode value, either "iptables" or "ipvs". Default is "iptables". See https://kubernetes.io/blog/2018/07/09/ipvs-based-in-cluster-load-balancing-deep-dive/ for further reference.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
 | outboundRuleIdleTimeoutInMinutes | no                        | Specifies a value for IdleTimeoutInMinutes to control the outbound flow idle timeout of the agent standard loadbalancer. This value is set greater than the default Linux idle timeout (15.4 min): https://pracucci.com/linux-tcp-rto-min-max-and-tcp-retries2.html                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| cloudProviderBackoff | no                        | Use the Azure cloudprovider exponential backoff implementation when encountering retry-able errors from the Azure API. Defaults to `true` for Kubernetes v1.14.0 and greater.                                                                                                                                                    |
```

Comment:
@CecileRobertMichon There was a rather large cloudprovider config documentation gap, so I used this PR as an excuse to backfill that info.